helpers/mathjax.js

```diff
@@ -21,6 +25,9 @@ export function loadMathJax(mathJaxConfig) {
 	if (mathJaxLoaded) return mathJaxLoaded;
 
 	window.MathJax = {
+		chtml: {
+			scale: (mathJaxConfig && mathJaxConfig.outputScale) || 1
```

Comment:
This has been supported in the LMS for some time now; it just wasn't propagated here originally.